tempodb/backend/azure/config.go

```diff
@@ -9,4 +13,5 @@ type Config struct {
 	Endpoint           string         `yaml:"endpoint-suffix"`
 	MaxBuffers         int            `yaml:"max-buffers"`
 	BufferSize         int            `yaml:"buffer-size"`
+	HedgeRequestsAt    time.Duration  `yaml:"hedge-requests-at"`
```

Comment:
Not specific to this PR: I noticed this config uses kebab-case instead of snake_case like the other configs. Is this intentional? Is this just some debt we can't get rid of anymore now?

Reply:
yeah, i noticed this for the first time adding this config option. i think everything is snake except for this azure config. we should consider a breaking change PR where we move azure to the same standard as everything else.